test/linter/test-browsers-data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const path = require('path');
+const chalk = require('chalk');
+const { Logger } = require('./utils.js');
+
+/**
+ * @typedef {import('../../types').Identifier} Identifier
+ * @typedef {import('../utils').Logger} Logger
+ */
+
+/**
+ * @param {Identifier} data
+ * @param {Logger} logger
+ * @returns {boolean}
+ */
+function processData(data, logger) {
+  const browser = Object.keys(data.browsers)[0];
```

Comment:
Why is only one browser tested?

Reply:
This test is run per each file, which only contains one browser!

Reply:
I see! But could we just loop over the browsers anyway so that it would also work if linting all of BCD?

Or if you'd prefer to keep it as is, a comment or an assert to explain?

Reply:
I've added a comment to explain the reasoning -- I'm expecting this to change once we get into the next-gen linter!